### ISSUE TITLE
G3 - Local search has no associated heading

### ISSUE THREAD

**Issue by jpinsent** [ccswbs/hjckrrh#204]
I noticed that for every local search input box that isn't on the main page, there is no heading associated with it. There appears to be an invisible h2 that has been commented out:

```
<!-- <h2 class="element-invisible"></h2> -->
```

Not sure if this was intentional or not. While not necessarily an accessibility issue, this is definitely a usability issue since both sighted and screen reader users have no indication that the input box is a local website search.

**Comment by mmafe** [comment]
This issue is related to https://github.com/brharp/hjckrrh/issues/212

We chose to remove the "Search" (screen-reader-only) heading that accompanied the search field. It was causing issues in Siteimprove because the first heading on the page was not a heading 1. We also noticed that neither WebAIM or SimplyAccessible.com are using the headings, so for now, we've commented it out.

Once issue #212 is resolved, users will have a better indication that the input box is a local website search. (Right now, it's only the placeholder text which is not enough.)
